frontend/src/container/FormAlertRules/ChartPreview/index.tsx

```diff
@@ -86,15 +94,24 @@ function ChartPreview({
 				],
 		  });
 
+	useEffect(() => {
```

Comment:
i guess moving to useQuery would help one

Reply:
useQuery is already being used.  Please share more details